parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -246,14 +245,15 @@ public <D> FileAppender<D> build() throws IOException {
       int rowGroupSize = context.rowGroupSize();
       int pageSize = context.pageSize();
       int pageRowLimit = context.pageRowLimit();
-      boolean dictionaryEnabled = context.dictionaryEnabled();
```

Comment:
can we still keep this in context ? it's private anyways and at times can help in overriding delete and data file seperatly, thoughts ?

Reply:
Agreed, I think it makes sense, it also just follows the existing pattern a lot better and is thus more readable.